creator-cards/amir-mokarchi.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "Amir Mokarchi",
+  "sectionTitle": null,
+  "countryCode": "IR",
+  "slogan": "",
+  "type": "CREATOR",
+  "bio": "Software Engineer | Microservices and Open Source Enthusiast",
+  "tags": ".NET, C#, SQL Server",
+  "socials": {
+    "youtube": "",
+    "linkedin": "https://linkedin.com/in/amir-mokarchi",
+    "twitter": "https://twitter.com/mokarchi",
+    "github": "https://github.com/mokarchi",
+    "mastodon": "",
+    "sessionize": ""
+  },
+  "channels": [
+    {
+      "name": "Blog",
+      "url": "https://medium.com/@mokarchi"
+    }
+  ],
+  "feeds": [{
+      "type": "Feed Blog",
```

Comment:
Hey @mokarchi this has to be either "dotnet" or "personal"

Dotnet -> only contains .NET related stuff (or Azure etc...)
Personal -> if your feed contains non programming related things or other languages like javascript frameworks etc.

Reply:
this feed only contains .NET and I modified with dotNet type